src/main/kotlin/com/openlattice/shuttle/MissionControl.kt

```diff
@@ -274,6 +274,7 @@ class MissionControl(
             propertyTypes = edmApi.propertyTypes.map { it.type to it }.toMap().toMutableMap()
             propertyTypesById = propertyTypes.mapKeys { it.value.id }
         } catch ( thrown: Throwable ) {
+            logger.error("Could not load EDM elements.  Check your authentication.")
```

Comment:
see: https://github.com/openlattice/shuttle/pull/160#discussion_r413359876

Reply:
Hm, it's in a try-catch with loading edm-elements, so the only things that can go wrong is "loading EDM elements" right?
And in my personal experience it's always an authentication issue.  Stackdown would have errored before.

Reply:
Oh, whoops, I didn't realize this was all API calls. I would get rid of `Check your authentication` just because that's too specific for the variety of errors that you could catch here. The `MissionControl.fail()` will log all of the info in `thrown` automatically, which should be enough for anyone debugging to tell precisely why it failed.